extra-doc/docbook-sgml/autobuild/defines

```diff
@@ -1,6 +1,6 @@
 PKGNAME=docbook-sgml
 PKGSEC=doc
-PKGDEP="sgml-common"
+PKGDEP="sgml-common docbook-dtd"
 PKGDES="Document type definitions for verification of SGML data files against the DocBook rule set"
```

Comment:
Best to mark this as a “transitional package for…,” same applies to others below.